Gorse.NET/Gorse.cs

```diff
@@ -21,6 +21,12 @@ public override string ToString()
     }
 }
 
+public class Neighbor
```

Comment:
Let us use `Score` as class name.

Reply:
public class Score
{
    public string UserId { get; set; }
    public double Score { get; set; }
}
C# doesn't allow members (properties, methods, etc.) to have the same name as the class.

Reply:
Should I use a JSON Property as follows

public class Score
{
public string UserId { get; set; }
[JsonProperty("Score")]
public double UserScore { get; set; }
}

Reply:
No worries, I've changed it back to UserScore!